app/controllers/specialist_mappers_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class SpecialistMappersController < ApplicationController
+  before_action :set_specialist_mapper, only: %i[show update destroy]
+
+  def index
+    json_response(SpecialistMapper.all)
```

Comment:
What if the Specialist is much, can we paginate the data with will_paginate or another gem

Reply:
Sure, Let me work on that.